torchrl/objectives/value/advantages.py

```diff
@@ -218,7 +218,7 @@ def set_keys(self, **kwargs) -> None:
                 and (value not in self.value_network.out_keys)
             ):
                 raise KeyError(
-                    f"value key '{self.tensor_keys.value}' not found in value network out_keys"
+                    f"value key '{value}' not found in value network out_keys"
```

Comment:
We could also add the values that would be accepted.
I usually have customized error messages such that users don't have to dig into the code to see where the typo is.

```suggestion
                    f"value key '{value}' not found in value network out_keys {self.value_network.out_keys}"
```